### PR TITLE
added get end point for crime and schools

### PR DESCRIPTION
**Pull request by krishnaramya**
please peer review.

### CODE REVIEW

File: src/HRQLS/Controllers/Schools/School.php [hunk 1]
```diff
@@ -75,8 +75,8 @@ public function main(Request $req, Application $app)
      */
     public function get(Request $req, Application $app, $id)
     {
-        //@TODO Krishna will finish implementing this.
-        return json_encode([ 'endpoint' => '/schools/{id}', 'id' => $id ]);
+        $esResult = $app['elasticsearch']->get($this->indices, $this->types, $id);
```
Comment (hop-along-polly) [inline src/HRQLS/Controllers/Schools/School.php]: This is absolutely wrong. School data is retrieved from the great schools API not Elasticsearch.

**Review by hop-along-polly — COMMENTED** [review]
Still needs unit tests for ElasticsearchServiceProvider::get method

File: src/HRQLS/Controllers/Schools/School.php (hunk 1 above)
Comment (krishnaramya) [reply src/HRQLS/Controllers/Schools/School.php]: I thought we are storing in Elastic search.

**Comment by hop-along-polly** [comment]
For crime yes...for schools no

**Comment by hop-along-polly** [comment]
Thinking a little more on this I don't think we should bother with the Get action for Schools. Alicia doesn't need it to populate data on the map and Great Schools already provides the functionality so us adding it here adds no value. Let me know your thought but come Sunday I'm planning on removing that section of this PR and just merging the crime stuff.

**Comment by asedarski** [comment]
Removing it sounds good to me

**Comment by krishnaramya** [comment]
if we are not going to populate, removing is fine.

**Pull request closed without merging**